socorro/external/es/super_search_fields.py

```diff
@@ -393,6 +393,23 @@ def get(self):
         "query_type": "enum",
         "storage_mapping": {"type": "long"},
     },
+    "bug_1541161": {
+        "data_validation_type": "str",
+        "description": (
+            "Dump JS Stack in case loadinfo is null within "
+            "NewChannelFromURIWithProxyFlagsInternal()."
+        ),
+        "form_field_choices": None,
+        "has_full_version": False,
+        "in_database_name": "Bug_1541161",
+        "is_exposed": True,
+        "is_returned": True,
+        "name": "bug_1541161",
+        "namespace": "raw_crash",
+        "permissions_needed": ["crashstats.view_pii"],
```

Comment:
This has stack traces, but it's not clear whether that's totally devoid of pii to me, so I'm going to make this require pii for now.